DuggaSys/tests/sectionedservice_Test.php

```diff
@@ -165,87 +165,58 @@
","pos":2,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-25 15:30:00","qrelease":"2015-01-08 00:00:00","feedbackenabled":0},{"entryname":"PHP Example 3","pos":3,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-20 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"Javascript examples","pos":4,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"New Code","pos":5,"kind":2,"visible":0,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"JavaScript Example 1","pos":5,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-18 15:30:00","qrelease":"2015-01-08 00:00:00","feedbackenabled":0},{"entryname":"JavaScript Example 2","pos":6,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-10 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"JavaScript Example 3","pos":7,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-15 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 examples","pos":8,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"HTML5 Example 1","pos":9,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-05 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 2","pos":10,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-20 15:30:00","qrelease":"2015-02-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 3","pos":11,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 4","pos":12,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 5","pos":13,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 6","pos":14,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 7","pos":15,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 8","pos":16,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0},{"entryname":"Shader examples","pos":17,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"Shaderprogrammering","pos":18,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0},{"entryname":"Shaderprogrammering","pos":19,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0}]}',
```

Comment:
In all cases there is two instances of linstentrie that shares the same pos. In the case above it is both New Code and Färgdugga 2 that has pos 5.

Reply:
Now, when running the test by itself it gives the same result every time and is always indexed from 1. This means that some other test is destroying the possibility to reproduce this test.

Reply:
I have run the tests one after the other to see what broke it and the problem is the reorderListentries test. It has to be reset to make the createListentrie test work.

Reply:
This is the correct expected output, this will only work the first time:
{"entries":[{"entryname":"PHP examples","pos":1,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"PHP Example 1","pos":2,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-30 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"PHP Example 2","pos":3,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-25 15:30:00","qrelease":"2015-01-08 00:00:00","feedbackenabled":0},{"entryname":"PHP Example 3","pos":4,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-20 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"New Code","pos":5,"kind":2,"visible":0,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"Javascript examples","pos":5,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"JavaScript Example 1","pos":6,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-01-18 15:30:00","qrelease":"2015-01-08 00:00:00","feedbackenabled":0},{"entryname":"JavaScript Example 2","pos":7,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-10 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"JavaScript Example 3","pos":8,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-15 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 examples","pos":9,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"HTML5 Example 1","pos":10,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-05 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 2","pos":11,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-20 15:30:00","qrelease":"2015-02-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 3","pos":12,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 4","pos":13,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 5","pos":14,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 6","pos":15,"kind":2,"visible":1,"highscoremode":0,"deadline":"2015-02-19 15:30:00","qrelease":"2015-01-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 7","pos":16,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0},{"entryname":"HTML5 Example 8","pos":17,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0},{"entryname":"Shader examples","pos":18,"kind":1,"visible":1,"highscoremode":0,"deadline":null,"qrelease":null,"feedbackenabled":0},{"entryname":"Shaderprogrammering","pos":19,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0},{"entryname":"Shaderprogrammering","pos":20,"kind":2,"visible":1,"highscoremode":0,"deadline":"2020-06-30 00:00:00","qrelease":"2020-05-01 00:00:00","feedbackenabled":0}]}

Reply:
I will make a new issue to reset the reorderListentrie after that test is done.